bitnami/postgresql/README.md

```diff
@@ -525,7 +525,7 @@ $ helm upgrade my-release stable/postgresql \
 
 > Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
 
-### To 10.0.0
+## To 10.0.0
```

Comment:
This is a level inside `## Upgrading` so it should contain 3#, check others
```suggestion
### To 10.0.0
```

Reply:
My bad, the error is on the "To 9.0.0" line.